CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
+* Print submission command queue property [\#7004](https://github.com/kokkos/kokkos/pull/7004)
+* SYCL: Add support for Graphs [\#6912](https://github.com/kokkos/kokkos/pull/6912)
+
+#### OpenMPTarget:
+* Honor user requested loop ordering in MDRange policy [\#6925](https://github.com/kokkos/kokkos/pull/6925)
+* Use mutex lock for guarding scratch space in parallel_scan. [\#6998](https://github.com/kokkos/kokkos/pull/6998)
+
+#### OpenACC:
+* Change the default execution behavior of the parallel_for(team-policy) constructs in the OpenACC backend [\#7012](https://github.com/kokkos/kokkos/pull/7012)
+
+#### HPX:
+* Workaround issue with template argument deduction to support compilation with NVCC [\#7015](https://github.com/kokkos/kokkos/pull/7015)
+
+### General Enhancements
+* Add converting constructor in `Kokkos::RandomAccessIterator` [\#6929](https://github.com/kokkos/kokkos/pull/6929)
+* Add `kokkos_swap(Array<T, N>)` specialization [\#6943](https://github.com/kokkos/kokkos/pull/6943)
+* Add a check precondition non-overlapping ranges for the `adjacent_difference` algorithm in debug mode [\#6922](https://github.com/kokkos/kokkos/pull/6922)
+* Improve compile-times when building with `Kokkos_ENABLE_DEBUG_BOUNDS_CHECK` in Cuda [\#7013](https://github.com/kokkos/kokkos/pull/7013)
+* Adding `Kokkos::to_array` [\#6375](https://github.com/kokkos/kokkos/pull/6375)
+* CTAD deduction guides for TeamPolicy [\#7030](https://github.com/kokkos/kokkos/pull/7030)
+* Cuda: Fix nvcc warnings [\#7021](https://github.com/kokkos/kokkos/pull/7021)
+* SIMD: Allow flexible vector width for 32 bit types [\#6802](https://github.com/kokkos/kokkos/pull/6802)
+* Host: Use `unlikely` attribute when reference counting views on host backends.[\#6730](https://github.com/kokkos/kokkos/pull/6730)
+* Make `Kokkos::Array` equality-comparable [\#7148](https://github.com/kokkos/kokkos/pull/7148)
+* Add nvidia Grace CPU architecture: `Kokkos_ARCH_ARMV9_GRACE` [\#7158](https://github.com/kokkos/kokkos/pull/7158)
+* Structured binding support for `Kokkos::complex` [\#7040](https://github.com/kokkos/kokkos/pull/7040)
+
+### Build System Changes
+* `develop` is the default branch on GitHub now.
+* Do not require OpenMP support for languages other than CXX [\#6965](https://github.com/kokkos/kokkos/pull/6965)
+* Update Intel GPU architectures in Makefile [\#6895](https://github.com/kokkos/kokkos/pull/6895)
+* Fix use of OpenMP with Cuda or HIP as compile language [\#6972](https://github.com/kokkos/kokkos/pull/6972)
+* Define minimum compiler versions for C++20 support [\#7128](https://github.com/kokkos/kokkos/pull/7128), [\#7123](https://github.com/kokkos/kokkos/pull/7123)
+
+### Incompatibilities (i.e. breaking changes)
+* Drop `Kokkos::Array` specialization of `Impl::ViewMapping` [\#6906](https://github.com/kokkos/kokkos/pull/6906)
+* Deprecate trailing `Proxy` template argument in `Kokkos::Array` [\#6934](https://github.com/kokkos/kokkos/pull/6934)
```

Comment:
Should this be moved to the `Deprecations` section?

Reply:
@Rombur thanks, addressed in 848fabe0114fee9a81eb3f5e793bc81157523416